src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -127,6 +127,21 @@ const RightBoxSection = ({ title, info, initialOpenState, afterTitle, onClick, c
   ])
 }
 
+const UnboundDiskNotification = () => {
+  return div({ style: Style.dashboard.rightBoxContainer }, [
+    div({ style: { padding: '0.5rem' } }, [
+      strong(['Persistent disks can no longer be shared between workspaces. ']),
+      h(Link, { href: Nav.getLink('environments'), style: { wordBreak: 'break-word' } }, [
+        'Click here'
```

Comment:
Can we make the link text something that makes more sense (for screen reader/accessibility)? Perhaps drop "Click here" and just make the link text "Review your shared disks and choose where each should go".

Reply:
Thanks - that's a good idea